src/app.ts

```diff
@@ -5,16 +5,28 @@ const passport = require('passport');
 const AuthStrategy = require('passport-http-bearer').Strategy;
 const amqp = require('amqplib/callback_api');
 const fs = require('fs');
+const helmet = require('helmet');
 
-import * as BodyParser from 'body-parser'
+import * as Cors from 'cors';
+import * as BodyParser from 'body-parser';
 import { Connection } from 'amqplib';
 import { Request, Response } from 'express';
 // Internal dependencies.
 import { GatewayMessageHandler } from './message_handling';
 
+const corsOptions: Cors.CorsOptions = {
+    origin: 'http://localhost:15300',
```

Comment:
Might be worth migrating to a config eventually (as per https://github.com/ents-crew/uems-gateway/issues/7#issuecomment-653784594) but this isn't a priority for now

Reply:
I agree